pandas/io/parquet.py

```diff
@@ -242,9 +269,11 @@ def read(
             mode="rb",
         )
         try:
-            result = self.api.parquet.read_table(
+            table = self.api.parquet.read_table(
                 path_or_handle, columns=columns, **kwargs
-            ).to_pandas(**to_pandas_kwargs)
+            )
+            result = table.to_pandas(**to_pandas_kwargs)
```

Comment:
Any reason to have `table` as an intermediary result instead of the original approach? I personally find the original implementation easier to read, being more compact.

Reply:
Yes, it is passed into `_read_attrs` below.